roles/openshift_openstack/tasks/provision.yml

```diff
@@ -14,6 +14,24 @@
   register: template_validation_output
   when: openstack_cli_exists|succeeded
 
+- name: Dry-run the stack
+  command: openstack stack create --dry-run -t {{ stack_template_path }} {{ openshift_openstack_stack_name }}
```

Comment:
This fails when you run the provision playbook for the second time (before removing it first).

`os_stack` does a `stack update` if the stack already exists. We have to do the same here (or disable the dry-run entirely, but I prefer the former).